core/services/aws/cognitoTrigger/lambdas/postConfirmation.ts

```diff
@@ -0,0 +1,42 @@
+import { DynamoDBClient } from '@aws-sdk/client-dynamodb'
+import { PutCommand } from '@aws-sdk/lib-dynamodb'
+import { PostConfirmationTriggerEvent, Callback, Context } from 'aws-lambda'
+import KSUID from 'ksuid'
+
+const dynamoDBClient = new DynamoDBClient({ region: process.env.AWS_REGION })
+
+// TODO: Refactor to separate the database logic from the Lambda handler for better separation of concerns.
```

Comment:
In my opinion, refactored version should be committed, rather than keeping it as a comment to be done in a later scope